Amplify/Categories/DataStore/Model/Internal/Schema/ModelField+Association.swift

```diff
@@ -91,13 +91,13 @@ public enum ModelAssociation {
     case hasOne(associatedFieldName: String?, targetNames: [String])
     case belongsTo(associatedFieldName: String?, targetNames: [String])
 
-    public static let belongsTo: ModelAssociation = .belongsTo(associatedFieldName: nil, targetNames: [])
+    public static let belongsTo: ModelAssociation = .belongsTo()
```

Comment:
let's revert this change as well since `case belongsTo(associatedFieldName: String?, targetNames: [String])` doesn't have default values for `associatedFieldName` and `targetNames`, we should keep the static method `.belongsTo` with the default values